server/handler/assignments.go

```diff
@@ -28,9 +28,13 @@ func GetAssignmentsForUserExperiment() RequestProcessFunc {
 		}
 
 		userID := service.GetUserID(r.Context())
-		assignments, err := repository.GetAssignmentsFor(userID, experimentID)
+		if userID == 0 {
```

Comment:
we do it often. How about moving it to a helper?
```
func checkUserInCtx(ctx) (id, error)
```

Reply:
done in c9d26ae